platforms/documentation/docs/src/snippets/plugins/dependenciesBlock/kotlin/buildSrc/src/main/kotlin/com.example.custom-dependencies.gradle.kts

```diff
@@ -0,0 +1,3 @@
+import com.example.ExamplePlugin
+
+apply<ExamplePlugin>()
```

Comment:
Can't we get rid of this file, and use the `java-gradle-plugin` to make this a purely Java example?

In our default (Kotlin) example, you have to understand 3 different languages to follow along - this makes the usage of these blocks seem much more complex than necessary.

Reply:
In both examples, there are only two languages (Kotlin/Java or Groovy/Java).  Where do you see 3?

Reply:
Ah, my mistake, this is fine.

Reply:
I removed the Kotlin/Groovy specific parts and there's now only Java and the Groovy/Kotlin usage examples